### ISSUE TITLE
Bump Chart, Sort streams by, Sort series by

### ISSUE THREAD

**Issue by vpallacci1** [rawgraphs/rawgraphs-app#271]
When you change "Sort streams by" or "Sort series by" option nothing happens. 

**In which chart does the bug happen?**
- Bump chart

**RAWGraphs Version**
Beta 2.0 (app version 24/03/21)

**Web browser and version**
Chrome 89.0.4389.90

**Operating System**
MacOS Big Sur 11.2.3

**Project file**
[Foreign Residents in Milan.rawgraphs.zip](https://github.com/rawgraphs/rawgraphs-app/files/6197456/Foreign.Residents.in.Milan.rawgraphs.zip)

**Additional context**

<img width="1351" alt="Schermata 2021-03-24 alle 14 14 36" src="https://user-images.githubusercontent.com/71371362/112316407-5f3c0f80-8cab-11eb-8a49-9c99fbc749fe.png">
<img width="1352" alt="Schermata 2021-03-24 alle 14 14 45" src="https://user-images.githubusercontent.com/71371362/112316411-6105d300-8cab-11eb-8947-77d18100c91f.png">

**Comment by mikima** [comment]
Yes that's true! this option doesn't make sense in this chart - it is an option of the area chart. Will remove it in the new version
